pages/help/help_domain-management.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Manage your .gov domain
+permalink: /help/domain-management
```

Comment:
Unfortunately I think you have to add a trailing `/` here or the link won't work correctly, so
`permalink: /help/domain-management/`